components/BarcodeScanner.vue

```diff
@@ -46,6 +48,7 @@ export default {
 
     // Constants
     let scanner = null;
+    const abortController = new AbortController();
```

Comment:
Note that if this component can do more than one scan (before it has to go out of scope or get cleaned up or whatever by Vue), then we'll need a new `AbortController` for each scan. Otherwise it will always abort after the first abort and never scan again.